lib/utils/exceptionHandler.js

```diff
@@ -0,0 +1,14 @@
+const { SoftError } = require('../errors')
+const logger = require('../logger')
+
+module.exports = (err) => {
+  if (!(err instanceof SoftError)) {
+    if (err.code && err.message) {
+      logger.error(`${err.message} (${err.code})`)
+    } else if (err.stack) {
+      err.stack = err.stack.split('\n').slice(0, 4).join('\n')
+      logger.error(process.env.LOG_LEVEL !== 'debug' ? err.message : err)
```

Comment:
This line will only be executed if the stack is set. Previously the logging will be executed anyway. Can you reverse it to the previous logic?